scripts/migrateHistoricBids.ts

```diff
@@ -0,0 +1,118 @@
+import * as fs from "fs";
+import * as env from "env-var";
+import * as dotenv from "dotenv";
+import * as mongodb from "mongodb";
+import { Bid } from "../src/types";
+import { MongoClientOptions } from "mongodb";
+import { queueAdapters, MessageQueueService } from "../src/messagequeue";
+import { CreateBatchOptions } from "@azure/event-hubs";
+
+import {
+  MessageType,
+  TypedMessage,
+  BidPlacedV1Data,
+  BidCancelledV1Data,
+} from "@zero-tech/zns-message-schemas";
+dotenv.config();
+
+const outputFilename = "./output/zAuctionBidsHistorical.json";
+const user = env.get("MONGO_USERNAME").required().asString();
+const pass = env.get("MONGO_PASSWORD").required().asString();
+const uri = env.get("MONGO_CLUSTER_URI").required().asString();
+const dbName = env.get("MONGO_DB").required().asString();
+const collectionName = env.get("MONGO_COLLECTION").required().asString();
+const fullUri = `mongodb+srv://${user}:${pass}@${uri}/`;
+const fileOption = process.argv[2] ?? false;
```

Comment:
consider just making this an env entry, e.g ALT_OUTPUT_DESTINATION. Or alternatively consider a package like [yargs](https://www.npmjs.com/package/yargs) which can really improve the readability of cli arguments in code.

Reply:
Added package yargs to simplify argv value retrieval